### ISSUE TITLE
"SpecViz" Viewer1d does not render correctly in web/Electron

### ISSUE THREAD

**Issue by havok2063** [spacetelescope/jdaviz#43]
Part of ASB Q1 work entails developing a modular 1-d spectral viewer component than can potentially be plugged into multiple ASB sites, using the jdaviz "specviz" Viewer1d app.  Displaying a spectrum currently works inside Jupyter notebooks but does not work inside the Electron app or any hosted website.  I have tried several Chrome and Firefox, a fresh computer, and a fresh install with up-to-date dependencies.  

If it works for others, can we get a list of the exact versions of all necessary software components?  

For ASB work to progress, this needs to be functionally stable for the foreseeable future.   Temporarily we can use an `ImageViewer` component or something like that so we're not entirely blocked by this.     

My `widget_code.json` is 
```
    "filename = '/Users/bcherinka/Work/notebooks/datb/manga-7495-12704-LOGCUBE.fits'",
    "from jdaviz.vizcomponents.viewer.viewer1d import Viewer1D",
    "from jdaviz.vizcomponents.vizapp import VizApp",
    "vizapp = VizApp()",
    "vizapp.glue_app.load_data(filename)",
    "v1d = Viewer1D(vizapp)",
    "v1d.show()"  
```
The web display and console traceback is attached below. 
<img width="675" alt="Screen Shot 2019-10-24 at 12 55 22 PM" src="https://user-images.githubusercontent.com/1836302/67507901-e2dc7180-f65d-11e9-8aa2-6898bdf9e8bb.png">

<img width="539" alt="Screen Shot 2019-10-24 at 12 55 38 PM" src="https://user-images.githubusercontent.com/1836302/67507906-e5d76200-f65d-11e9-9f9f-1327e162c40c.png">

**Comment by nmearl** [comment]
Underlining codebase has changed drastically. Please open a new ticket if seeing something similar in the new jdaviz.
